### ISSUE TITLE
Source fields as free form text fields

### ISSUE THREAD

**Issue by patrickdrouin** [gaois/terminologue#88]
Hello,

is there any way sources could be offered as free form text fields? I understand this would have quite an impact on the existing structure, but it would be useful for teaching. 

Thanks,
Patrick

**Comment by michmech** [comment]
Not at the moment, but it is doable in principle. I think people should be able to specify the source of an example either by selecting a source from a list (as now), or by typing into a free text box (if you don't want to bother creating a list), or both: selecting a source from a list and then giving some additional details (page number, URL) as free text. This would entail adding one more data field to the entry schema and that's not a big deal, it wouldn't break anything.

**Comment by patrickdrouin** [comment]
Yes, a system that lets you choose between a list or free text would be amazingly useful.
